src/misc/lv_obj_property.h

```diff
@@ -0,0 +1,134 @@
+/**
+ * @file lv_obj_property.h
+ *
+ */
+
+#ifndef LV_OBJ_PROPERTY_H
+#define LV_OBJ_PROPERTY_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if LV_USE_OBJ_PROPERTY
```

Comment:
You should include `lv_conf_internal.h` first (or an other header file which indirectly includes it) to see if `LV_USE_OBJ_PROPERTY` is set or not.